tests/grid/GridViewJqueryClientScriptTest.php

```
@@ -0,0 +1,291 @@
+            ],
+            $gridView->clientScript->getClientOptions($gridView),
+            "Should replace default selector with 'overrideFilterSelector'.",
+        );
+    }
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $_SERVER['REQUEST_URI'] = 'https://example.com/';
+    }
```

PHPMD (2.15.0)

[error] setUp accesses the super-global variable $_SERVER. (undefined)

(Superglobals)